Automattic/wp-calypso
# Unify calypso user registration complete

#### Changes proposed in this Pull Request

Unifies `calypso_user_registration_complete`.

#### Testing instructions

* Test normal registration, passwordless registration, social registration
* Make sure `flow` and `type` props are present and properly set in the `calypso_user_registration_complete` Tracks event.

Fixes #
